src/lib/stringIndices.js

```diff
@@ -1,16 +1,19 @@
-  let stringIndices = (userString, userNumber) => {
+
+let stringIndices = (userString, userNumber) => {
     let newArr = []
     let separate = ""
+    let emptyString = " "
```

Comment:
Consistent indentation should be used in your code.  Best practice is a two space indentation.